configparser.go

```diff
@@ -200,3 +196,55 @@ func (p *ConfigParser) SaveWithDelimiter(filename, delimiter string) error {
 
 	return nil
 }
+
+// ParseReader parses data into ConfigParser from provided reader.
+func (p *ConfigParser) ParseReader(in io.Reader) error {
```

Comment:
Not 100% sure if this func should be public. But since we have `New`, `NewWithOptions` and `NewWithDefaults` I think it is the right thing to do